bedrock/firefox/templates/firefox/australis/tour-base.html

```diff
@@ -23,8 +23,6 @@
 
 {% block body_class %}sky{% endblock %}
 
-{% block site_css %}{% endblock %}
```

Comment:
Ah, also, `firefox/australis/firstrun-tour.html` doesn't load `tabzilla-static.less`. Maybe another reason to add it to the `firefox` LESS bundle?